portaljs-pages/page-admin-process-visu/resources/page.properties

```diff
@@ -1,5 +1,5 @@
 displayName=Bonita Admin Process Visualization
 name=custompage_adminProcessVisuBonita
-description=Default admin process visualization
+description=This page provides detail information about a process visualization. It is dedicated to Admin-like profiles. Default admin process visualization. This page has not been created with the UI Designer.
```

Comment:
```suggestion
description=This page provides the BPMN visualization and execution data of all open cases of a process. It is dedicated to Admin-like profiles. This page has not been created with the UI Designer but it can still be used as a custom page in any application.
```